cryptofeed/exchanges/ascendex_futures.py

```diff
@@ -0,0 +1,55 @@
+import re
+from collections import defaultdict
+from typing import Dict, Tuple
+
+from cryptofeed.connection import RestEndpoint, Routes, WebsocketEndpoint
+from cryptofeed.defines import ASCENDEX_FUTURES, L2_BOOK, TRADES, PERPETUAL
+from cryptofeed.exchanges import AscendEX
+from cryptofeed.symbols import Symbol
+
+
+# noinspection PyAbstractClass
+class AscendEXFutures(AscendEX):
+    """
+    Docs, https://ascendex.github.io/ascendex-futures-pro-api-v2/#introducing-futures-pro-v2-apis
+    """
+    id = ASCENDEX_FUTURES
+    websocket_channels = {
+        **AscendEX.websocket_channels,
+    }
+    # Docs, https://ascendex.github.io/ascendex-futures-pro-api-v2/#how-to-connect
+    # noinspection PyTypeChecker
+    websocket_endpoints = [WebsocketEndpoint(
+        'wss://ascendex.com:443/api/pro/v2/stream',
+        channel_filter=(
+            websocket_channels[L2_BOOK],
+            websocket_channels[TRADES],
```

Comment:
why channel filters?

Reply:
As they have private endpoint for private stream, it is for futures private channels support.

See, https://ascendex.github.io/ascendex-futures-pro-api-v2/#how-to-connect